scripts/auth-open.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
> none of the test utils is executable

Better check that again, there are 5 executables that have been living there for quite a while
https://github.com/dcos/dcos/blob/master/test_util/run-all is one such example.

A recommended name might be be something like "get_dcos_oauth_ci_acs_token.py"